index.bs

```diff
@@ -168,16 +161,16 @@ Compute the Layout Shift {#sec-compute-layout-shift}
 <div algorithm="compute the layout shift">
     When asked to to <dfn export>compute the layout shift</dfn>, with <var>unstable elements</var>, a list of impacted <a>element</a>'s on the page, as input, run the following steps:
 
-    1. Let <var>impacted region</var> be initially set to empty.
+    1. Let <var>impacted region</var> be initially set to <code>null</code>.
     1. Let <var>max shift fraction</var> be initially set to 0.
     1. For each tuple <var>tuple</var> in <var>unstable elements</var>:
         1. Take the geometric union of <var>impacted region</var> with <var>tuple</var>’s <a>currentVisualRepresentation</a> and <var>tuple</var>’s <a>previousVisualRepresentation</a> if <a>previousVisualRepresentation</a> is not null.
```

Comment:
This step seems a bit unclear. It is meant to change impacted region, right? So I think it start like `If <a>previousRepresention</a> is not null, set |impacted region| to...`. Alternatively, we call an algorithm (if it's defined elsewhere) which specifies how to compute the geometric union, and go from that.

Reply:
Done.